site/docs/components/drawer.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: Drawer
+navTitle: Drawer
+summaryParagraph: A drawer is a temporary workspace that allows users to complete tasks without navigating to a new page.
+tags: ["Modal", "Modal window", "Slide-out modal", "Menu", "Compartment", "Dossier", "Overlay", "Lightbox"]
+needToKnow:
+- Drawers come in two sizes, Small (375px max-width) and Medium (600px max-width). Choose the right one for your needs. The horizontal width of content should always fit the drawer. Users should never be able to scroll horizontally within a drawer. 
+- Drawers can have overlays or can appear in parallel with page content. An overlay drawer is useful when you want to focus the user on a specific task. A parallel drawer can provide additional context or information during a task. 
+
+Status: Beta component
+---
+
+import WhenToUseAndWhenNotToUse from "docs-components/WhenToUseAndWhenNotToUse"
+import WhenToUse from "docs-components/WhenToUse"
+import WhenNotToUse from "docs-components/WhenNotToUse"
+
+## Visuals
+
+### UI Kit
+
+<iframe style="border: 1px solid rgba(0, 0, 0, 0.1);" width="800" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FGMxm8rvDCbj0Xw3TQWBZ8b%2FZen-UI-Kit%3Fnode-id%3D13514%253A75372" allowfullscreen></iframe>
+
+### Examples
+
+<iframe style="border: 1px solid rgba(0, 0, 0, 0.1);" width="800" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FGMxm8rvDCbj0Xw3TQWBZ8b%2FZen-UI-Kit%3Fnode-id%3D13514%253A75611" allowfullscreen></iframe>
+
+## To keep in mind
+
+### Medium vs Small
+
+**Use Medium Drawers (600 px):**
+*   When you need to display additional or supplemental page content
+*   When you want to provide additional navigation within the Drawer such as [Tabs](/components/tabs/).
+
+**Use Small Drawers (375 px):**
+*   When it’s important to stay in context and when the action applies changes to the parent page.
+*   When you expect this functionality to be required on mobile. Because a small drawer does not need to rearrange or re-flow like a page or modal does, it is often a friendlier experience on mobile. 
```

Comment:
It's not a big deal, but I'm not sure "re-flow" is accurate here. This usually refers to when browsers need to re-draw the page due to the user changing the window size (if the content changes this can also trigger a reflow/redraw). But you wouldn't ever change the window size on mobile so it wouldn't need to reflow technically speaking (it would just render once). I guess what you're saying is that a small drawer will look the same on mobile as it will be the same width, whereas a medium one would get squashed?